test/unit_test/api/db/services/test_pipeline_operation_log_service_create.py

```
@@ -0,0 +1,434 @@
+        FAIL = "fail"
+
+    sys.modules["common.constants"].PipelineTaskType = PipelineTaskType
+    sys.modules["common.constants"].TaskStatus = TaskStatus
+    sys.modules["api.db"].VALID_PIPELINE_TASK_TYPES = {
+        PipelineTaskType.PARSE, PipelineTaskType.RAPTOR, PipelineTaskType.GRAPH_RAG,
+    }
+
+    # Load the actual module under test.
+    import importlib.util
+    import os
+    os.chdir("/tmp/opencode/repos/ragflow")
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/opencode/repos/ragflow"

(S108)